projects/1.18/assets/tetra/tetra/lang/zh_cn.json

```
@@ -39,315 +42,346 @@
   "tetra.holo.craft.modular_shield": "盾牌",
   "tetra.holo.craft.modular_crossbow": "弩",
   "tetra.holo.craft.slot": "插槽",
-  "tetra.holo.craft.material_requirement": "可以由 %s",
+  "tetra.holo.craft.material_requirement": "可以由%s制作",
   "tetra.holo.craft.improvements": "改进",
-  "tetra.holo.craft.improvement_button": "%s改进",
+  "tetra.holo.craft.improvement_button": "%s种改进",
   "tetra.holo.craft.stats": "属性",
   "tetra.holo.craft.variants": "变种",
-  "tetra.holo.craft.variants_filter": "Input a string to filter out modules based on their name, prefix with # to filter based on material category.\n \n§7[§8f§7] §8focus\n§7[§8enter§7] §8toggle focus",
-  "tetra.holo.craft.variants_sort": "Sort all variants based on the selected stat\n \n§7[§8s§7] §8toggle dropdown\n§7[§81§7] - §7[§89§7] §8select stat",
+  "tetra.holo.craft.variants_filter": "根据名字过滤组件，在最前面加上“#”以根据材料类型过滤\n \n§7[§8f§7] §8选中搜索框\n§7[§8enter§7] §8切换是否选中搜索框",

```

```suggestion
  "tetra.holo.craft.variants_filter": "根据名字过滤组件，在最前面加上“#”以根据材料类型过滤\n \n§7[§8f§7] §8选中搜索框\n§7[§8Enter§7] §8切换是否选中搜索框",

```
没找到游戏内在哪里，但是大写估计还是得要的

---

这个在全息球每个部件的页面里
左上角有几个按钮啥的，其中一个的提示文本